rocAL/include/pipeline/master_graph.h

```diff
@@ -100,6 +100,9 @@ class MasterGraph {
     Status run();
     Timing timing();
     RocalMemType mem_type();
+    RocalBatchPolicy last_batch_policy();
+    bool last_batch_padded();
```

Comment:
may be better to change the function name to is_last_batch_padded() to make it more clear?